src/bot.py

```
@@ -31,23 +33,35 @@
-    )
+    get_context_start_time = time.perf_counter()
+    messages: list[dict[str, object]] = session.get_context(
+        tokens=1024,  # Can edit this to be anything
+        peer_target=peer.id,
+    ).to_openai(assistant=assistant)
+
     messages.append({"role": "user", "content": prompt})
 
+    get_context_end_time = time.perf_counter()
+    get_context_duration = get_context_end_time - get_context_start_time
+    logger.info(f"Got context from Honcho in {get_context_duration:.2f} seconds")
```

Ruff (0.13.3)

Logging statement uses f-string

(G004)